tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -86,6 +103,12 @@ def model_to_dot(model,
     expand_nested: whether to expand nested models into clusters.
     dpi: Dots per inch.
     subgraph: whether to return a `pydot.Cluster` instance.
+    layer_range: range of layers passed as `list` or `Tensor` of `int` or `str`
```

Comment:
Why should we support tensor type here? Since Tensor could be graph tensor whose value is not available until evaluated, should we just support the raw python type here, eg list of int or str?

Reply:
done